### DISCUSSION TITLE
essay/2024/11/20/find-side-project-coworker

### DISCUSSION THREAD

**Discussion by giscus** [BenjaminKim/benjaminkim.github.io#245] [Comments]
# essay/2024/11/20/find-side-project-coworker

사이드 프로젝트 개발자를 구합니다. 기획은 80% 이상 완료되어 있는 상태입니다. 현재 팀 구성원 : 기획자, 디자이너, 마케터. 플러터와 백엔드 개발을 다루는 풀스택 개발자를 찾습니다. 보상: 처음에는 드릴 수 없으나 수익이 나면 공평하게 분배해 드립니다. 원할 시 스톡옵션도 드릴 수 있습니다.

https://jeho.page/essay/2024/11/20/find-side-project-coworker.html

**Comment by ncrash** [comment]
전형적인 재주는 곰이 부리고 느낌이 강하게 드는 구인(?)글이군요

아이디어는 있는데 의지가 약해서 진도가 나가질 않는 분들이 이런걸 활용하셔서 아이디어를 검증(?)해보고 적당할때 엑싯(?)하면 괜찮지 않나 싶네요

AI 도구들이 워낙 좋아 날개를 단 요즘이니 이전처럼 갈아넣지 않아도 결괴물을 빠르게 만들 수 있게 되었으니요

**Comment by sunhohong** [comment]
여기에서 제일 불안한 포인트는 기획 80% 완료네요. 가보면 아무것도 없음 😅

**Reply by BenjaminKim** [reply]
으하하, 저도 같은 생각입니다.

**Comment by h1ro21** [comment]
댓글달려고 가입까지했네요... 내 생에 깃텁에 가입을 할 줄이야...
기획퍼센테이지.....구성원 ㅋㅋㅋ....아 내 글인줄ㅋㅋㅋㅋ
같이 사이드 하고 있는 디자이너한테 해당 글 링크 받고 한참을 웃었네요ㅋㅋ....그리고 한참을 찝찝해한.... 후우

제가 딱 그렇거든요.
사이드 다 모집하고 프론트(풀스택) 혼자 하기엔 너무 딜레이 될거 같아서 백엔드 사이드 모집중인...

물론 뭐 기재하신 케이스도 많이 있겠지만 허허....
그리고 이런 글을 썼다는건 나름 좀 치여(?)보셔서 그런 것도 있을 수 있겠지만
안그런 케이스도 있다는 것도 알아주셨으면 해요 :)

만들고 싶으면 혼자 만든다 = 아무리 기술이 발전하고 편해졌다해도 
사람마다 특화된 강점이 다르듯 어려운 사람은 분명 존재합니다. 
하지만 일반적인 개발자 시각에서는 생각할 수 없는 사업아이템이라던지 
서비스운영, 영업력, 아이디어, 디자인 등등이 있을 수는 있죠.

그리고 한편으로는 개발자 중에 단순히 아이디어 혹은 구현할 능력만 있지 
UXUI는 커녕 디자인 감각도 없고 그것을 사업으로 끌어가기 어려운 사람이 있을수도 있구요. 

(물론 그게 다 되는 개발자라면야 굳이 이런 글을 보기도 전에 이미 몇십~몇백개의 서비스를 런칭해보고 실패해보고 성공해보고 해봤겠죠?) 

그런 사람들에게는 사이드프로젝트를 통해서 새로운 기회나 인연, 경험을 해보는것도 
좋은 터닝포인트가 될 수도 있다고 봐요.

무튼 너무 그렇게 부정적인 시각으로만 보지말아주세요 :)
현명한 사람이라면야 부정적인 상황에서도 무언가를 배우고 얻어갈테니까요.

**TMI긴하지만 저는 이미 MVP 1차 기획 끝났고 디자인도 1차끝났어요😂😂아무것도없진않아욬ㅋㅋㅋ

**Reply by BenjaminKim** [reply]
말씀하신 내용 다 동감합니다.
다만, 제가 쓴 글의 요지는 보상에 대한 부분입니다.
h1ro21님도 본문의 내용처럼 개발자에게 보상을 하고 계신건가요?

**Reply by h1ro21** [reply]
보상? 글 요지에 의하면 금전적인 것을 뜻하는 것으로 보이네요. 맞나요?
그렇다면 당연히 당장엔 보상은 안/못하고 있네요ㅎㅎ
 
그런데 사이드란게 보통 그런거 아닌가요?
단순히 금전적인 보상을 떠나서 자기에게 어떠한 형태로든 득이된다면 조인하고 진행.

인맥? 재미? 역할? 배움? 가능성? 새로움?
참여에 이유가 있다면 조인하는거라 생각합니다. 그 이유가 보상으로 여겨질 수 있다면 보상은 보상이겠네요.

지금 제 프로젝트에 조인된 사람들 다 각자 업계 8~15년 차 + 시리즈d~pre-IPO, 대기업 등등 다양하게 섞여있는데 금전적보상? 같은 얘기 누구 하나 꺼내지 않습니다. 차피 메인 잡이 있고 말그대로 "사이드"인 만큼, 다른 무언가가 그들을 설득했다고 보여지네요. (고맙...)

오히려 제가 추가적인 동기부여를 위해 구성원 각자의 입장에 따른 리워드를 주려고 생각중입니다. (스탁옵션은 아닙니다....이건 진짜 선넘긴했네요) 가까울수록 이런건 확실히 해야하거든요ㅎ

**Reply by BenjaminKim** [reply]
그렇다면 제가 글에 적은 내용과는 다른 이야기네요.
단순히 재미로 사이드 프로젝트 하는 걸 나쁘게 볼 사람은 없을 것 같습니다.
제가 개발자 동생이 있다면 재미와 공부 삼아 다른 팀원들과 사이드 프로젝트 해보라는 조언을 해줄 수 있을 것 같습니다.
하지만 다른 사람들이 사이드 프로젝트의 이익을 거의 다 가져가는 경우라면 권하지 않을 겁니다.
